tests/dataproxy-engine-type-and-mongodb-datasource-provider.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { PrismaTemplates } from '~/src'
```

Comment:
This test suite shows that multiple preview flags append like we expect them to.